src/utils/useOnlineStatus.js

```diff
@@ -0,0 +1,19 @@
+import {useState, useEffect} from 'react';
+
+const useOnlineStatus = () => {
+    const [onlineStatus, setOnlineStatus] = useState(true);
+
+    useEffect(() => {
+        window.addEventListener("offline", () => {
+            setOnlineStatus(false);
+        });
+
+        window.addEventListener("online", () => {
+            setOnlineStatus(true);
+        });
+    }, [])
+
+    return onlineStatus
+}
+
+export default useOnlineStatus;
```

Comment:
A custom hook to alter the display based on user device connection status to Network